panels/info/cc-info-panel.c

```diff
@@ -1840,6 +1843,24 @@ cc_info_panel_init (CcInfoPanel *self)
       return;
     }
 
+  self->priv->hbox1 = WID ("hbox1");
+  self->priv->detail_vbox = WID ("detail_vbox");
+
+  if (gdk_screen_width () < 1024 || gdk_screen_height () < 600)
```

Comment:
I don't think the control center (differently from the settings daemon) has the ability to add third-party plugins. Either way, we don't ship them in Endless so it wouldn't be a concern.
It should be fine to add a method to the CcWindow API to retrieve the "small window" value.
Also yeah feel free to expand what the "small-window" property checks for, if what's there is not enough (but I'm curious to understand why it's not enough in that case).